packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -258,11 +263,12 @@ describe('MenuTrigger', function () {
     ${'MenuTrigger'} | ${MenuTrigger} | ${{onOpenChange}}
   `('$Name does not trigger on disabled button', async function ({Component, props}) {
     let tree = renderComponent(Component, props, {}, {isDisabled: true});
-    let button = tree.getByRole('button');
-    await user.click(button);
+    let {setElement, getTrigger, getMenu} = testUtilUser.createTester('MenuTester');
+    setElement(tree.container);
+    await user.click(getTrigger());
     act(() => {jest.runAllTimers();});
```

Comment:
open?

Reply:
So I didn't use `open` because it will throw because it expects there to be an `aria-controls` after going through the interactions to open the menu (which it won't have because the menu is disabled) but this test is explicitly testing that the menu won't happen. Any opinions on if we should tone down when the errors are thrown? Maybe I should add an option to inform `open` that we expect the menu NOT to open?